src/main/java/seedu/duke/Main.java

```diff
@@ -24,37 +103,67 @@ public static void main(String[] args) {
         WeekView weekView = new WeekView(startOfWeek, dateFormatter);
         TaskManager taskManager = new TaskManager();
 
+        boolean printWeek = true; // Flag to control printing of the week view
+        boolean inMonthView = false; // Flag to indicate if we are in month view mode
```

Comment:
It might be better to rename these booleans to sound more "boolean-like". For example, we could use the names isPrintingWeek and isInMonthView respectively.